apps/src/sites/studio/pages/courses/index.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import CourseOverview from '@cdo/apps/templates/courseOverview/CourseOverview';
+
+$(document).ready(showCourseOverview);
+
+function showCourseOverview() {
+  const courseData = document.querySelector('script[data-coursedata]');
+  const configCourse = JSON.parse(courseData.dataset.coursedata);
```

Comment:
I'm not sure what `configCourse` means (the configured course?) and `courseData.dataset.coursedata` is not particularly helpful.  Can we work out some nicer names here?

Can we recover more gracefully if `JSON.parse` fails here?
![image](https://cloud.githubusercontent.com/assets/1615761/25872334/247ce6c8-34bf-11e7-9f34-c4f715995721.png)